memory_core/memory_core_magma.py

```diff
@@ -67,27 +67,26 @@ def __init__(self,
                  banks=1,
                  input_iterator_support=6,  # Addr Controllers
                  output_iterator_support=6,
-                 input_config_width=16,
-                 output_config_width=16,
+                 config_width=16,
                  interconnect_input_ports=2,  # Connection to int
                  interconnect_output_ports=2,
                  mem_input_ports=1,
                  mem_output_ports=1,
-                 use_sram_stub=True,
+                 use_sim_sram=True,
                  sram_macro_info=SRAMMacroInfo("TS1N16FFCLLSBLVTC512X32M4S",
                                                wtsel_value=0, rtsel_value=1),
                  read_delay=1,  # Cycle delay in read (SRAM vs Register File)
                  rw_same_cycle=False,  # Does the memory allow r+w in same cycle?
                  agg_height=4,
-                 tb_sched_max=16,
                  config_data_width=32,
                  config_addr_width=8,
                  num_tiles=1,
                  fifo_mode=True,
                  add_clk_enable=True,
                  add_flush=True,
                  override_name=None,
-                 gen_addr=True):
+                 gen_addr=True,
+                 GF=True):
```

Comment:
This seems weird. sram_macro_info still specifies the TSMC sram, but GF is set to True. What does this actually do? Seems like instead of having a GF boolean, it would be better to just specify which technology/macro is being used.

Reply:
I agree about the GF boolean...in fact, Alex, didn't you already create a parameter for this, something like `tech=gf8` or some such thing?

Reply:
No I haven't.

Reply:
Fixed to use a string?